openedx/features/content_tagging/api.py

```diff
@@ -0,0 +1,91 @@
+"""
+Content Tagging APIs
+"""
+from typing import Generator, List, Type
+
+import openedx_tagging.core.tagging.api as oel_tagging
+from openedx_tagging.core.tagging.models import Taxonomy
+from organizations.models import Organization
+
+from .models import TaxonomyOrg
+
+
+def create_taxonomy(
+    name: str,
+    org_owners: List[Organization] = None,
+    description: str = None,
+    enabled=True,
+    required=False,
+    allow_multiple=False,
+    allow_free_text=False,
+    system_defined=False,
```

Comment:
@pomegranited We should not be able to create system defined in this generic api. In my PR I could create another way to create them more consistently